test/release/examples/patterns/readcsv.chpl

```diff
@@ -0,0 +1,286 @@
+/* This example demonstrates how to read in a CSV file.
+
+   The example csv file is from Kaggle,
+   https://www.kaggle.com/allen-institute-for-ai/CORD-19-research-challenge?select=metadata.csv
+   The included metadata-20lines.csv only has 20 lines of data including the
+   header row that has the column names.
+
+   Each line in csv files is a row of data.  Columns are separated by commas.
+   Some of the cells of data have quotes because the data string itself includes
+   a comma or commas.
+
+   There are many options in terms of data structures that such data could be
+   read into.  Here are the two approaches illustrated here.
+       (1) pythonic -- list of Map instances where there is an entry in the Map
+               per column (key=column name, value=column value for that row)
+       (2) chapeltastic -- associative array with the key being the column name
+               and the value being an array of strings
+
+   You can compile and run this example by on the command line doing a
```

Comment:
"by on"

I also wonder whether you should start with building Chapel here. i.e. you can also put a link to the building chapel documentation, for example https://chapel-lang.org/docs/master/usingchapel/building.html

Reply:
Yeah I was surprised as well, but it is a 4-minute run without --fast and less than a second with --fast.

I fixed the "by on" and put in a sentence about building Chapel with the link you suggested.